### ISSUE TITLE
Ignition fails creating folders in /sysroot

### ISSUE THREAD

**Issue by arithx** [openshift/os#123]
As a result of running `kola run` against RHCOS I've found that Ignition is failing when attempting to create directories inside of `/sysroot`.

```
[    8.262414] Starting Setup Virtual Console...ignition[734]: CRITICAL : files: createFilesystemsFiles: createFiles: op(4): [failed]   writing file "/resource/data": mkdir /sysroot/resource: permission  denied
```

I'm unsure if this is something that's intended.

**Comment by dustymabe** [comment]
`/etc/` and `/var/` (and things bind mounted under `/var`) are the only read-write filesystems. so I would say "by design"

**Comment by arithx** [comment]
Added https://projects.engineering.redhat.com/browse/COREOS-185 to update all kola tests to stop attempting to write to the root of the filesystem.

**Comment by miabbott** [comment]
This was fixed with coreos/mantle#882 and coreos/mantle#883
